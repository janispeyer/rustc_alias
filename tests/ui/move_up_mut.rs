// compile-flags: -Zmir-emit-retag

fn move_up_mut(x: &mut i32, y: &mut i32) -> i32 {
    *x = 42;
    *y = 7;
    return *x;
}

fn main() {
    let mut x = 1;
    let mut y = 2;
    move_up_mut(&mut x, &mut y);
}
