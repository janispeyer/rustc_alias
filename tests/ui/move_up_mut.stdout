main: done
move_up_mut: done
